packages/orbit-components/src/primitives/IllustrationPrimitive/index.js

```diff
@@ -48,14 +48,17 @@ const IllustrationPrimitive = ({
   size = SIZE_OPTIONS.MEDIUM,
   dataTest,
   spaceAfter,
-}: Props) => (
-  <StyledImage
-    illustrationName={name}
-    alt={alt || name}
-    size={size}
-    data-test={dataTest}
-    spaceAfter={spaceAfter}
-  />
-);
+}: Props) => {
+  const getAltText = alt === "" || (alt && alt.length > 0) ? alt : name;
```

Comment:
I'm having a hard time reading this logic. Could you write a `getAltText` function and use `if` statements to make the intent clearer?